### PR TITLE
Add docker-compose & fix boto3 version

### PR DESCRIPTION
**Pull request by leezu**

### CODE REVIEW

File: tools/jenkins-slave-creation-unix/conf-ubuntu-cpu-c5/install.sh [hunk 1]
```diff
@@ -51,8 +51,8 @@ sudo -H -S -u jenkins_slave git config --global user.email "mxnet-ci"
 sudo -H -S -u jenkins_slave git config --global user.name "mxnet-ci"
 
 #Install python3, pip3 and dependencies for auto-connect.py
-sudo apt-get -y install python3 python3-pip python3-boto3 python3-jenkins python3-joblib
-sudo pip3 install "docker<4.0.0"
+sudo apt-get -y install python3 python3-pip python3-yaml python3-jenkins python3-joblib
+sudo pip3 install "docker<4.0.0" boto3
```
Comment (ChaiBapchya) [inline tools/jenkins-slave-creation-unix/conf-ubuntu-cpu-c5/install.sh:55]: I looked up diff between `apt-get install python3- ` vs `pip3 install ` here : https://askubuntu.com/questions/431780/apt-get-install-vs-pip-install
but couldn't quite get why we need to do this for boto3
Shouldn't both work?

Comment (ChaiBapchya) [reply tools/jenkins-slave-creation-unix/conf-ubuntu-cpu-c5/install.sh:55]: As an after thought `conf-ubuntu-gpu/install.sh` uses `pip3 install `. But still curious

Comment (leezu) [reply tools/jenkins-slave-creation-unix/conf-ubuntu-cpu-c5/install.sh:55]: boto3 in ubuntu 18.04 repo is too old and doesn't support secretsmanager.

Comment (leezu) [reply tools/jenkins-slave-creation-unix/conf-ubuntu-cpu-c5/install.sh:55]: This bug was introduced in my previous PR

**Pull request merged**